pyquil/api/wavefunction_simulator.py

```diff
@@ -0,0 +1,156 @@
+import warnings
+
+from six import integer_types
+
+from pyquil.paulis import PauliSum
+from pyquil.quil import Program
+from pyquil.wavefunction import Wavefunction
+from ._base_connection import ForestConnection
+
+
+class WavefunctionSimulator:
+    """
+    Represents a connection to the QVM.
+    """
+
+    def __init__(self, connection: ForestConnection, random_seed=None):
+        """
+        Constructor for wavefunction simulation. Sets up any necessary security.
+
+        :param sync_endpoint: The endpoint of the server for running small jobs
```

Comment:
These params are all wrong/copied pasted

Reply:
updated docs for new classes `WavefunctionSimulator`, `QPU`, and `QVM`.